gadgets/mri_core/NoiseAdjustGadget.cpp

```diff
@@ -64,6 +65,10 @@ namespace Gadgetron{
     perform_noise_adjust_ = this->get_string_value("perform_noise_adjust")->size() ? this->get_bool_value("perform_noise_adjust") : true;
     GDEBUG("NoiseAdjustGadget::perform_noise_adjust_ is %d\n", perform_noise_adjust_);
 
+    str = this->get_string_value("pass_nonconformant_data");
```

Comment:
This is too complicated:

    pass_nonconformant_data_ = this->get_bool_value("pass_nonconformant_data");

Should suffice